lasio/writer.py

```diff
@@ -43,6 +44,10 @@ def write(
             be estimated from the first two rows of the first column.
         fmt (str): Python string formatting operator for numeric data to be
             used.
+        column_fmt (dict or None): use this to set a different format string
+            for specific columns from the data ndarray. E.g. to use ``'%.4f'``
+            for the depth column and ``'%.2f'`` for all the other columns,
+            you would use ``fmt='%.2f', column_fmt={0: '%.3f'}``.
```

Comment:
How would you write different formats for different columns? 
i.e. in the example here is '%.3f' for depth and '%.2f' for the rest. 
What if you want to format depth '%.3f', second column{1} '%.4f', third column{2} '%.3f', and rest of data '.%2f'. Could you please give an example?

Reply:
``las.write(fp, fmt='%.2f', column_fmt={0: '%.3f', 1: '%.4f', 2: '%.3f'})``